### PR TITLE
Dronov's First Goodie

### PR DESCRIPTION
**Pull request by dronov**
# Goodie Pull Request Template

Using this template will help us better understand your Instant Answer and assist you when necessary.  Simply copy and paste the markdown below into the description of your GitHub pull request and complete as appropriate.

**What does your Instant Answer do?**

Just saying a few words about Mikhail Dronov when user entered "duckduckhack dronov" at the search input on the DuckDuckGo webpage.

**What problem does your Instant Answer solve (Why is it better than organic links)?**

It's educational Goodie.
**What is the data source for your Instant Answer? (Provide a link if possible)**

You can find that the Mikhail Dronov is real by looking at the website: http://dronov.net

**Why did you choose this data source?**

Cause I started write to dronov.net five years ago.

**Are there any other alternative (better) data sources?**

Yeah, just look on the social links at http://dronov.net

**What are some example queries that trigger this Instant Answer?**

"duckduckhack dronov"

**Which communities will this Instant Answer be especially useful for? (gamers, book lovers, etc)**

This answer may be useful for beginers at duckduckhack.

**Is this Instant Answer connected to a DuckDuckHack [Instant Answer idea](https://duck.co/ideas)?**

I'm not sure. I think no. But if it will be necessary I'm available to do it.

**Which existing Instant Answers will this one supercede/overlap with?**

Mmm.. Maybe "duckduckhack *"

**Are you having any problems? Do you need our help with anything?**

Could you suggest me useful Perl resources to learn essential things about Perl and some resources where I can find the best practices? Thank you.

**Where did you hear about DuckDuckHack? (For first time contributors)**

I found DuckDuckHack at the Internet. I found it the great opportunity to make an influence on modern Internet. It's awesome!

**What does the Instant Answer look like? (Provide a screenshot for new or updated Instant Answers)**

## Checklist
Please place an 'X' where appropriate.

```
[x] Added metadata and attribution information (https://duck.co/duckduckhack/metadata)
[x] Wrote test file and added to t/ directory (https://duck.co/duckduckhack/goodie_tests)
[x] Verified that instant answer adheres to design guidelines (https://duck.co/duckduckhack/design_styleguide)
[x] Verified that instant answer adheres to code styleguide (https://duck.co/duckduckhack/code_styleguide)
[x] Tested cross-browser compatibility

    Please let us know which browsers/devices you've tested on:
    - Windows 8
        [] Google Chrome
        [] Firefox
        [] Opera
        [] IE 10

    - Windows 7
        [] Google Chrome
        [] Firefox
        [] Opera
        [] IE 8
        [] IE 9
        [] IE 10

    - Windows XP
        [] IE 7
        [] IE 8

    - Mac OSX
        [] Google Chrome
        [] Firefox
        [] Opera
        [] Safari

 - CentOS 7
        [x] Firefox
        [x] Chrome

    - iOS (iPhone)
        [x] Safari Mobile
        [] Google Chrome
        [] Opera

    - iOS (iPad)
        [] Safari Mobile
        [] Google Chrome
        [] Opera

    - Android
        [] Firefox
        [x] Native Browser
        [] Google Chrome
        [] Opera
```

### CODE REVIEW

File: lib/DDG/Goodie/IsAwesome/Dronov.pm [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+topics "geek", "programming";
+code_url "https://github.com/duckduckgo/zeroclickinfo-goodies/blob/master/lib/DDG/Goodie/IsAwesome/Dronov.pm";
+attribution github => ["dronov", "Mikhail Dronov"],
+            twitter => "dronovmm";
+
+triggers start => "duckduckhack dronov";
+
+handle remainder => sub {
+    return if $_;
+	return "Mikhail Dronov is awesome and has successfully completed the DuckDuckHack Goodie tutorial! He's saying hello from far cold Russia. Find him at dronov.net";
```
Comment (MrChrisW) [inline lib/DDG/Goodie/IsAwesome/Dronov.pm]: Can you fix the indentation here, this should be 4 spaces (soft tabs) as per the [code style guide](https://duck.co/duckduckhack/code_styleguide)

**Comment by MrChrisW** [comment]
@dronov Поздравляю (congratulations) on your first Goodie :cake: I've added a little feedback, once fixed we'll get it merged and live!

**Comment by dronov** [comment]
@MrChrisW Thank you! I've fixed it. The Codio's default editor is a little bit strange and didn't want to show me error in indentation.

File: lib/DDG/Goodie/IsAwesome/Dronov.pm [hunk 2]
```diff
@@ -0,0 +1,28 @@
+package DDG::Goodie::IsAwesome::Dronov;
+# ABSTRACT: Dronov's first Goodie
+# Start at https://duck.co/duckduckhack/goodie_overview if you are new
+# to instant answer development
```
Comment (mintsoft) [inline lib/DDG/Goodie/IsAwesome/Dronov.pm]: Can you delete this comment and just the ABSTRACT comment above?

File: lib/DDG/Goodie/IsAwesome/Dronov.pm [hunk 3]
```diff
@@ -0,0 +1,28 @@
...
+
+use DDG::Goodie;
+
+zci answer_type => "is_awesome_dronov";
+zci is_cached   => 1;
+
+name "Mikhail Dronov";
+description "http://dronov.net";
+primary_example_queries "first example query", "second example query";
```
Comment (mintsoft) [inline lib/DDG/Goodie/IsAwesome/Dronov.pm]: Can you fill out the primary example queries and remove the optional secondary ones below?

**Comment by MrChrisW** [comment]
@dronov Thanks for updating, merged! :+1: This should be live shortly.

**Comment by moollaza** [comment]
@dronov it's live! https://duckduckgo.com/?q=duckduckhack+dronov&t=ffcm&ia=answer

**Pull request merged**